Mailer/app/api/Validation/InputValidatorTrait.php

```diff
@@ -29,17 +32,23 @@ protected function validateInput(string $request, string $schema)
         $schema = file_get_contents(__DIR__ . '/schemas/' . $schema);
         $data = Json::decode($request);
         $validator = new Validator();
-        $validator->validate($data, (object) $schema);
+        $validator->validate($data, (object) json_decode($schema));
 
         if (!$validator->isValid()) {
             $data = ['status' => 'error', 'message' => 'Payload error', 'errors' => []];
             foreach ($validator->getErrors() as $error) {
-                $data['errors'][] = "{$error['property']}: {$error['message']}";
+                $this->errors[] = "{$error['property']}: {$error['message']}";
             }
-            Debugger::log("Cannot parse paywall content data - " . Json::encode($payload) . ' -> ' . implode(', ', $data['errors']));
+            $data['errors'] = $this->errors;
+            Debugger::log("Cannot parse paywall content data - " . Json::encode($payload) . ' -> ' . implode(', ', $this->errors));
```

Comment:
This doesn't look like it belongs here.

Reply:
I just copied this part from previous use.

Reply:
We'll change the other implementation too; the message should be generic and reference only payload parsing.